job/src/test/java/com/bazaarvoice/emodb/job/service/TestDefaultJobService.java

```diff
@@ -53,7 +52,7 @@ public void setUp() throws Exception {
         LifeCycleRegistry lifeCycleRegistry = mock(LifeCycleRegistry.class);
         _queueService = mock(QueueService.class);
         _jobHandlerRegistry = new DefaultJobHandlerRegistry();
-        _jobStatusDAO = new InMemoryJobStatusDAO();
+        JobStatusDAO jobStatusDAO = new InMemoryJobStatusDAO();
```

Comment:
Same thing about changing naming convention just for one variable.

Reply:
see comment above